students/tri_nguyen/lesson05/mailroom.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python3
+
+# ------------------------------------------- #
+# Student:   Tri Nguyen
+# Instructor: Natasha Aleksandrova
+# Date:  09-Mar-2018
+# ------------------------------------------- #
+
+# Import stuff
+from operator import itemgetter
+import time
+
+# Get local time
+local_time = time.localtime()
+
+# Declare variables and data structure
+
+donor_dict = { #  [0] == total donation, [1] == number of gifts, [2] == average
+
+    'William Gates, III': [653784.49, 2, 326892.24],
+    'Mark Zuckerberg': [16396.10, 3, 5465.37],
+    'Jeff Bezos': [877.33, 1, 877.33],
+    'Paul Allen':[708.42, 3, 236.14],
+    'Tri Nguyen': [100.00, 1, 100.00]
+    }
```

Comment:
for a little more clarity it would be a good idea to use a dict for the value as well, like so:
`{"Jeff Bezos": {"total": 877.33, "num_of_gifts": 3, "avg": 877.33}, }`
your code is now well documented and easy to understand.